dev-ruby/travis/travis-1.8.13.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+USE_RUBY="ruby23 ruby24 ruby25"
```

Comment:
Please update the ruby targets. ruby23 is obsolete and should be removed. ruby26 and ruby27 should be tried to see if the package is ready for them (as determined by dependencies, test suite and changelog).